data/implementations/record_stores.json

```
@@ -1,17 +1,13 @@
 {
   "id": "record-stores",
-  "interface": "https://github.com/libp2p/interface-record-store",
+  "title": "Storage",
+  "interface": "https://github.com/libp2p/js-libp2p-interfaces",
   "libs": [
     {
       "id": "record",
```

```suggestion
      "id": "libp2p-record",
```
Any reason this doesn't follow the naming convention of others?